app/v1/controllers/RegisterNinoController.scala

```diff
@@ -20,10 +20,11 @@ import javax.inject.{Inject, Singleton}
 import org.slf4j.MDC
 import play.api.Logger
 import play.api.libs.json.{JsValue, Json}
-import play.api.mvc.{Action, AnyContent, ControllerComponents, Result}
+import play.api.mvc._
```

Comment:
Can you remove the unused imports from this file please

Reply:
Sure thing.